debianize.sh

```diff
@@ -70,15 +70,17 @@ elif [[ "$ACTION" == "D" || "$ACTION" == "d" ]]; then
 	echo "So, please copy the already compiled: airspaceconverter libairspaceconverter.so and airspaceconverter-gui binaries to this folder."
 	read -p "Press ENTER when done"
 
+	if [ "$1" != "test" ]; then
```

Comment:
Come mai il parametro test?

Reply:
può essere utile compilare e di seguito generare i pacchetti prendendo i binari dalla directory Release e buildQT al posto che dalla home del progetto (utile invece quando arrivano da altra distro). Con lo stesso parametro saltavo anche la cancellazione dei binari da Release e buildQT, utili per testarli in locale.